### ISSUE TITLE
Feature request: allow regex for "good-names"

### ISSUE THREAD

**Issue by lucatrv** [PyCQA/pylint#3086]
As in subject, for instance:

```python
good-names=df.*
```

to accept all variable names beginning with "df".

**Comment by PCManticore** [comment]
Sounds like a good suggestion, thank you!
